pmda/density.py

```diff
@@ -259,10 +256,15 @@ def __init__(self, atomgroup, delta=1.0, atomselection=None,
         elif not updating and atomselection is not None:
             raise ValueError("""With updating=False, the atomselection='{}' is
                         not used and should be None""".format(atomselection))
+        elif updating and atomselection is not None:
+            self._select_atomgroup = atomgroup.select_atoms(atomselection,
+                                                            updating=True)
```

Comment:
Do updating AtomGroups work with the serialization?

Reply:
Yes! Thanks to what has already been implemented by Richard:)